pandera/config.py

```diff
@@ -32,6 +32,7 @@ class PanderaConfig:
     export PANDERA_VALIDATION_DEPTH=DATA_ONLY
     export PANDERA_CACHE_DATAFRAME=True
     export PANDERA_KEEP_CACHED_DATAFRAME=True
+    export PANDERA_MAX_FAILURE_CASES=100  # Default; use -1 for unlimited
```

Comment:
Is this syntax consistent with other libraries? (The name is imprecise - as if pandera will only validate 100 cases, whereas pandera is actually validating all...)

_(I couldn't find anything in pydantic or logfire...)_